front/pages/w/[wId]/builder/data-sources/managed.tsx

```diff
@@ -360,11 +368,20 @@ function getTableColumns() {
       meta: {
         width: "6rem",
       },
-      cell: (info: CellContext<RowData, number | null>) => (
+      sortingFn: (rowA: Row<RowData>, rowB: Row<RowData>) => {
+        return (
+          (rowA.original.managedDataSource.usage?.count ?? 0) -
+          (rowB.original.managedDataSource.usage?.count ?? 0)
+        );
+      },
```

Comment:
I think you can remove the sortingFn if you have an accessorKey like `managedDataSource.usage.count` (assuming usage is never null, which seems to be the case)

Reply:
I'm newb with tanstack, will try. Thanks !